### PR TITLE
Tests lists

### PR DESCRIPTION
**Pull request by imobach**
This PR is just a WIP and, by the way, it deserves more discussion. I'm replacing our regular test RSpec examples with *shared examples*. Why?

* Integration with OpenQA, by SLEPOS team, will be a lot easier as they will "only" need to retrieve a list of tests (`sles12.list`, for example) and run every script listed there.
* For us is less repetitive (DRY). If we need something special, we can also add an `it` block as usual.

Anyway, I want to discuss this proof-of-concept with @schubi2.

### CODE REVIEW

**Comment by schubi2** [comment]
Hey, I like the concept. Please go on !

File: test/spec_helper.rb [hunk 1]
```diff
@@ -38,3 +38,15 @@
     end
   end
 end
+
+RSpec.shared_examples "test_scripts" do |name|
+  list_path =  File.join(File.dirname(__FILE__),"../test", "#{name}.list")
+
+  File.readlines(list_path).map(&:chomp).each do |line|
+    test, description = line.split("#")
```
Comment (lslezak) [inline test/spec_helper.rb]: This split does not work correctly for descriptions containing `#`,  e.g. with `bnc#xxxxxx`, use `split("#", 2) to limit it to max. 2 parts.

Comment (imobach) [inline test/spec_helper.rb]: @lslezak Yes, you're right. I'll take that into account.

**Comment by schubi2** [comment]
LGTM

**Comment by imobach** [comment]
Thanks. Let's see if Jenkins is happy with this change :)

**Pull request merged**